configs/sst_kernel_rats-kernel-rt-base.yaml

```diff
@@ -5,11 +5,6 @@ data:
     description: Base set of kernel-rt packages and tools to ship
     maintainer: sst_kernel_rats
     packages:
```

Comment:
This field is missing value. You can do `packages: []` to leave it empty.

Reply:
Didn't realize that I needed the empty brackets.   I Hate YAML (tm)